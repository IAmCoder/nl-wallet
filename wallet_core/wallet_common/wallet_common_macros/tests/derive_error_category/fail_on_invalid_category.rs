#[derive(wallet_common_macros::ErrorCategory)]
enum Error {
    #[category(invalid)]
    Invalid,
}

fn main() {}
