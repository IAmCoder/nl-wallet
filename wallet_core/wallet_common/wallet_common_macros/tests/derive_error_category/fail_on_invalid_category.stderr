error: Expected any of ["expected", "critical", "pd", "defer"], got "invalid".
 --> tests/derive_error_category/fail_on_invalid_category.rs:3:16
  |
3 |     #[category(invalid)]
  |                ^^^^^^^
